pkg/service/users/users_service_impl.go

```diff
@@ -0,0 +1,41 @@
+package users_service
+
+import (
+	"pkg/service/pkg/data/request"
+	"pkg/service/pkg/data/response"
+	"pkg/service/pkg/models"
+	repository "pkg/service/pkg/repository/users"
+)
+
+type UsersServiceImpl struct {
```

Comment:
as mentioned - you can remove the Impl

Reply:
I cannot name it only UserService, as I have a struct with that name